code/UI/World/EntityWorldPanel/GrubWorldPanel.razor

```diff
@@ -25,6 +25,10 @@
     public GrubWorldPanel(Grub grub) : base(grub, Vector3.Up * 50f)
     {
         _health = Grub.Health;
+
+        // Enlarge the panel bounds to fit larger names.
+        // The default is ( -500, -500, 1000, 1000 )
```

Comment:
@trundlr How about I remove line 30 just because only setting the Panel size isn't necessarily obvious that it's larger than the default value because the default is never explicitly stated in the documentation for `RootPanel`.

Reply:
For sure!